instrumentation/couchbase/couchbase-2.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/couchbase/v2_0/CouchbaseUtil.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.couchbase.v2_0;
+
+import com.couchbase.client.core.metrics.DefaultLatencyMetricsCollectorConfig;
+import com.couchbase.client.core.metrics.DefaultMetricsCollectorConfig;
+import com.couchbase.client.java.cluster.BucketSettings;
+import com.couchbase.client.java.env.DefaultCouchbaseEnvironment;
+import java.util.concurrent.TimeUnit;
+
+public class CouchbaseUtil {
+
+  public static DefaultCouchbaseEnvironment.Builder envBuilder(
```

Comment:
this class is duplicated for 2.6 because the return types of methods in the builder class have changed, class compiled agains older version of the library won't work with newer version